parts/common/components.json

```diff
@@ -648,6 +645,10 @@
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes-csi/livenessprobe",
           "latestVersion": "v2.14.0"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes-csi/livenessprobe",
+          "latestVersion": "v2.15.0"
```

Comment:
This means we'll be caching both 2.14 and 2.15 of liveness prob:
* Do we need both of them? They are 164MB each on windows
* If the 2.14.0 is only needed while RP is rolled out to use 2.15 (due to caching) then we should use "previousLatestVersion" for 2.14 (as opposed to a seperate block). This indicates to the next developer that they might be able to remove 2.14 rather than keep it.

Reply:
RP will use 2.15 on March release, and then I will remove liveness and node-driver-registrar old versions